install/index.php

```diff
@@ -1864,34 +1862,14 @@ function revertSetting(defval, inpid)
 		{
 			$hostname .= ':'.$_SERVER['SERVER_PORT'];
 		}
-		$websiteurl = $hostname.'/';
-
+		
 		$currentlocation = get_current_location();
-		if($currentlocation)
-		{
-			// TODO: Change this to find the last position of /install/
-			$pos = my_strpos($currentlocation, '/install/');
-			if($pos === 0)
-			{
-				$cookiepath = "/";
-			}
-			else
-			{
-				$cookiepath = my_substr($currentlocation, 0, $pos).'/';
-			}
-		}
-
-		$currentscript = $hostname.get_current_location();
-
-		if($currentscript)
-		{
-			$bburl = my_substr($currentscript, 0, my_strpos($currentscript, '/install/'));
-		}
-
-		if($_SERVER['SERVER_ADMIN'])
-		{
-			$contactemail = $_SERVER['SERVER_ADMIN'];
-		}
+		$noinstall = substr($currentlocation, 0, strrpos($currentlocation, '/install/'));
```

Comment:
Actually it may, depending on the folder the user has put the install folder. I could put MyBB in folder téstèfõldër for example - unless I'm missing something.